includes/class-sensei-course.php

```diff
@@ -2860,29 +2857,6 @@ public static function load_single_course_lessons_query() {
 
 	}//end load_single_course_lessons_query()
 
-	/**
-	 * Flush the rewrite rules for a course post type
-	 *
-	 * @since 1.9.0
-	 *
-	 * @param $post_id
-	 */
-	public static function flush_rewrite_rules( $post_id ) {
```

Comment:
We should probably deprecate this function rather than remove it outright, as there was no `@access` tag to indicate privacy.

Reply:
I've deprecated them, but I've removed functionality. 48d037892adf701f5c46339fc71c7a3cdfb12e0d